indigo_app/templates/indigo_api/work_new_batch.html

```diff
@@ -184,22 +184,14 @@ <h4 class="card-header">
       <div class="card-body">
         Instructions:
         <ul>
-          <li>Make a copy of the <a href="https://docs.google.com/spreadsheets/d/1JJKal7IFjFKKojsHIJAW12elDUNIwAuMrCV4sJ1jUOc">Work info template</a>: 'File' | 'Make a copy…'</li>
-          <li>Make it accessible: 'SHARE'  | 'Get shareable link' (top-right corner)</li>
-          <li>Watch <a href="https://www.youtube.com/watch?v=tfNm_PtZ7JQ&index=2&list=PL7MJ_sFHs953q_23TdDhnvQWqxAkrc672">this video</a> for instructions on how to populate the spreadsheet</li>
-          <li>Choose a country from the drop-down</li>
-          <li>Paste the url for your spreadsheet into the box below that</li>
-          <li>Click on 'Import works'!</li>
+          <li>Make a copy of the <a href="https://docs.google.com/spreadsheets/d/1JJKal7IFjFKKojsHIJAW12elDUNIwAuMrCV4sJ1jUOc" target="_blank">Work info template</a>: 'File | Make a copy'.</li>
+          <li>Record the Spreadsheet URL under <a href="{% url 'place_settings' place=place.place_code %}" target="_blank">Settings</a>.</li>
+          <li>Share the spreadsheet with (give Edit rights to) indigo@indigo-laws-africa.iam.gserviceaccount.com.</li>
```

Comment:
You'll need to pluck this email address from the bulk_creator helper object, since it depends on the installation.

To be truly accurate, you should adjust this description based on whether the bulk creator's gsheets_api_enabled (or something similar) is True. If it's False, then fall back to the description for public sharing and CSV.